package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "orc-shared",
-	"version": "1.0.0-pre.5",
```

Comment:
For develop, it is suppose to be something like ""version": "1.1.0-dev.1". maybe you did not merge develop lately.